tests/regression_tests/torus/test.py

```diff
@@ -0,0 +1,39 @@
+import openmc
+import pytest
+
+from tests.testing_harness import PyAPITestHarness
+
+
+@pytest.fixture
+def model():
+    model = openmc.Model()
+    fuel = openmc.Material()
+    fuel.set_density('g/cm3', 12.0)
+    fuel.add_nuclide('U235', 1.0)
+    al = openmc.Material()
+    al.set_density('g/cm3', 1.0)
+    al.add_nuclide('H1', 1.0)
+    model.materials.extend([fuel, al])
+
+    # 🍩🍩🍩
```

Comment:
nice use of doughnuts

Reply:
That _is_ why unicode was invented, right? So we can put donuts in our code :smile:

Reply:
Wow :open_mouth: I guess you could say unicode's applications have come... full circle